bio_curve_fit/logistic.py

```diff
@@ -321,13 +321,32 @@ def predict_inverse(self, y):
         But for samples of unknown concentration, we want to get the concentration as given
         response, which is what this function does.
 
+        args:
+        y: float
```

Comment:
specify type that y should be (if it can be float or list, specify and check for it)